### PR TITLE
Start page

### PR DESCRIPTION
**Pull request by radzak**

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -51,6 +52,6 @@
       To begin the development, run `npm Start` or `yarn Start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  </body>
+</body>
 
-</html>
+</html>
```
Comment (gradam) [inline client/public/index.html]: no newline at the end of the file

File: client/src/views/Layout/components/Header/Toolbar/NavItems/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          Test
+        </NavLink>
+      </li>
+      <li>
+        <NavLink
+          exact
+          to="/login"
+          className={cx('navLink')}
+          activeClassName={cx('navLink--active')}
+        >
```
Comment (gradam) [inline client/src/views/Layout/components/Header/Toolbar/NavItems/index.js]: shouldn't it be in the previous line?

File: client/src/views/Layout/index.js [hunk 3]
```diff
@@ -7,22 +9,25 @@ import styles from './styles.css';
...
-          <main className={cx('content')}>{this.props.children}</main>
-          <div className={cx('footer')}>
-            <Footer/>
-          </div>
-        </div>
-      </Fragment>
-    );
-  }
-}
+const Layout = props => (
```
Comment (gradam) [inline client/src/views/Layout/index.js]: it is recommended to use () in arrow functions so `const Layout = (props) => ...`

File: client/src/views/Layout/index.js [hunk 4]
```diff
@@ -7,22 +9,25 @@ import styles from './styles.css';
...
+  </div>
+);
 
-export default Layout;
+Layout.propTypes = {
+  children: PropTypes.element.isRequired,
+  isAuthenticated: PropTypes.bool.isRequired
+};
+
+const mapStateToProps = state => ({
```
Comment (gradam) [inline client/src/views/Layout/index.js]: ditto

File: client/src/views/Layout/components/Header/index.js [hunk 5]
```diff
@@ -1,21 +1,25 @@
...
-};
+import Toolbar from './Toolbar';
+import styles from './styles.css';
+
+const cx = classNames.bind(styles);
 
-const mapStateToProps = state => ({
-  isAuthenticated: state.auth.authenticated
-});
+const Header = props => (
```
Comment (gradam) [inline client/src/views/Layout/components/Header/index.js]: it is recommended to use () in arrow functions so `const Header = (props) => ...`

File: client/src/views/Layout/index.js (hunk 3 above)
Comment (radzak) [reply client/src/views/Layout/index.js]: https://github.com/airbnb/javascript#arrows--one-arg-parens

File: client/src/views/Layout/index.js (hunk 4 above)
Comment (radzak) [reply client/src/views/Layout/index.js]: https://github.com/airbnb/javascript#arrows--one-arg-parens

File: client/src/views/Layout/components/Header/index.js (hunk 5 above)
Comment (radzak) [reply client/src/views/Layout/components/Header/index.js]: https://github.com/airbnb/javascript#arrows--one-arg-parens

File: client/src/views/Layout/components/Header/Toolbar/NavItems/index.js (hunk 2 above)
Comment (radzak) [reply client/src/views/Layout/components/Header/Toolbar/NavItems/index.js]: "it", what are you referring to?

File: client/public/index.html (hunk 1 above)
Comment (radzak) [reply client/public/index.html]: Oooops, sorry, but vs code default settings are soooo coool ...
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

File: client/src/views/Layout/components/Header/Toolbar/NavItems/index.js (hunk 2 above)
Comment (gradam) [reply client/src/views/Layout/components/Header/Toolbar/NavItems/index.js]: >

Comment (radzak) [reply client/src/views/Layout/components/Header/Toolbar/NavItems/index.js]: Nah, it's good. It's much clearer imo + I comply to the [airbnb](https://github.com/airbnb/javascript/blob/master/react/README.md) style guide.

File: client/public/index.html (hunk 1 above)
Comment (radzak) [reply client/public/index.html]: One of us is gonna change it after we make some changes to index.html. My editor already adds it automatically on save, so I won't forget about it.

**Pull request merged**